app/Http/Controllers/Consumables/ConsumableCheckoutController.php

```diff
@@ -69,6 +70,11 @@ public function store(Request $request, $consumableId)
             'note' => $request->input('note'),
         ]);
 
+        $available= new Consumable();
```

Comment:
Actually, I'm a little unclear why we need the new Consumable bit here? Wouldn't we want the num remaining on the consumable in question, not a new one?